src/SessionMonitor/index.js

```diff
@@ -7,6 +7,7 @@ export class SessionMonitor {
   constructor(updateSessionTime, inactiveTimeLimit) {
     this.updateSessionTime = updateSessionTime || 5 * 60 * 1000;
     this.inactiveTimeLimit = inactiveTimeLimit || 30 * 60 * 1000;
+    this.inactiveWorkspaceTimeLimit = workspaceTimeoutInMinutes * 60 * 1000 || 30 * 60 * 1000;
```

Comment:
probably can get rid of the later `30 * 60 * 1000` since `workspaceTimeoutInMinutes` will already have a default value